src/main/java/wolox/training/models/User.java

```diff
@@ -31,6 +34,8 @@
 @Getter
 @Entity(name = "USERS")
 @NoArgsConstructor
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@DiscriminatorColumn(name="USER")
```

Comment:
Sorry, I can't see any column named user 😅 maybe we should add a new column to the User model ?

Reply:
Done :D